front/app/components/ReactionControl/ReactionButton.tsx

```diff
@@ -436,7 +440,7 @@ const ReactionButton = ({
               icon={buttonReactionModeIsActive ? 'check' : 'vote-ballot'}
               bgColor={buttonReactionModeIsActive ? colors.success : undefined}
               className="e2e-ideacard-vote-button"
-              aria-describedby={disabledReason ? describedById : undefined}
```

Comment:
If aria-describedby's value is undefined, it won't be added to our HTML, so we can actually do this check for describedById, which also makes our code more readable.